### DISCUSSION TITLE
表元数据缓存为什么要设置个过期时间

### DISCUSSION THREAD

**Discussion by Mxgxg** [apache/incubator-seata#4983] [Q&A] [answered]
我看服务中已经有了每过60S去验证一次表元数据了，现在的情况是我不会去经常改表结构，即使改了也要重启服务，所以为了降低业务耗时，我在想能不能去掉这个过期时间。
![image](https://user-images.githubusercontent.com/79822553/194749686-9a77b6e1-bf10-4231-b185-bd46eb4a5d3c.png)

**Comment by funky-eyes** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/19943636/196350697-a13ce6b6-b31d-4e20-b88e-1e3cbf5fe7e6.png)
请不要关闭client.rm.tableMetaCheckEnable 并建议将client.rm.tableMetaCheckerInterval 设置低于60000L来避免这种情况

**Reply by Mxgxg** [reply]
不好意思，描述错了。我没想关闭tableMetaCheckEnable，我想问的是图中TABLE_META_CACHE这个缓存为什么要设置过期时间

**Reply by funky-eyes** [reply]
> TABLE_META_CACHE

其实不需要设置，我们准备直接使用ConcurrentHashMap了
